sp/src/game/server/ez2/npc_crabsynth.h

```diff
@@ -0,0 +1,238 @@
+//=============================================================================//
+//
+// Purpose:		Crab Synth NPC created from scratch in Source 2013 as a new type
+//				of Combine armored unit, filling a role somewhere between a standard
+//				soldier and a strider.
+// 
+//				This was originally created for Entropy : Zero 2.
```

Comment:
I think you should specify that it was made for the episodes / post EZ2 content, to avoid confusion that it was made for the main game. I'm not sure if the assassins and husks make that distinction

Reply:
This distinction is for the situation in which this NPC may be ported to other, non-Entropy : Zero mods in the future, such as the Map Labs mod template. Labeling it as being made for the episodes/post-E:Z2 content also feels a little backwards to me, but I guess I could look at specifying it as something like "Entropy : Zero 2 mod content".

Reply:
I totally understand - this label is supposed to mean "created for Entropy : Zero 2" as opposed to "created for MapLabs" or "created for Entropy : Zero Uprising".

But the problem is that in this repo, people will misread it to mean it was originally created for EZ2 and then cut or something. Because obviously it isn't in the EZ2 that was released. It's for EZ2, but after EZ2 - for the post release content. We can workshop the phrasing but I think it's important to clarify somehow because I am very confident readers will misunderstand.